source/docs/software/networktables/nt4-migration-guide.rst

```diff
@@ -0,0 +1,165 @@
+Migrating from NetworkTables 3.0 to NetworkTables 4.0
+=====================================================
+
+NetworkTables 4.0 (new for 2023) has a number of significant API breaking changes from NetworkTables 3.0, the version of NetworkTables used from 2016-2022.
+
+NetworkTableEntry
+-----------------
+
+While ``NetworkTableEntry`` can still be used (for backwards compatibility), users are encouraged to migrate to use of type-specific Publisher/Subscriber/Entry classes as appropriate, or if necessary, ``GenericEntry``. It's important to note that unlike ``NetworkTableEntry``, these classes need to have appropriate lifetime management. Some functionality (e.g. persistent settings) has also moved to ``Topic`` properties.
+
+NT3 code (was):
```

Comment:
@Daltz333 is there a way to put the code examples side by side on wider screens?

Reply:
Use `sphinx_design` or `sphinx_panels` for a dynamic grid

Reply:
> @Daltz333 is there a way to put the code examples side by side on wider screens?

Like https://discord.com/channels/176186766946992128/368993897495527424/797728025113788468 ?

Reply:
Unfortunately, it doesn't look good due to the small (1000px) horizontal width of the website.